doc/guacamole-playback-example/src/main/webapp/index.html

```diff
@@ -28,10 +28,19 @@
     <body>
 
         <!-- Guacamole recording player -->
-        <div class="player">
+        <div id="player">
```

Comment:
Eh ... now that the example is referring to the player as a single, distinct object within the JS, that would require doing something like `getElementsByClassName('player')[0]`, which is semantically wrong.

It might make sense to apply the player functionality as a behavior to absolutely all elements having the "player" class, but that would make the example overly complex IMHO.